.github/workflows/build.yml

```diff
@@ -181,14 +184,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '^1.15'
+          go-version: '^1.17'
       - uses: actions/setup-python@v2
         with:
           python-version: '3.7'
       - name: Install dependencies
         run: |
           pip install wheel
           go get -v -t ./...
+      - name: Update golang.org/x/sys (temporary, see https://github.com/golang/go/issues/45702)
```

Comment:
This should be put into go.mod, not the CI pipeline if possible.

Reply:
Agreed! Realised this right after I made this change. Will push one more commit.